incubator/piwik/templates/_helpers.tpl

```diff
@@ -0,0 +1,24 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 24 -}}
```

Comment:
we can now use longer names.
Base this file on the contents of https://github.com/kubernetes/charts/blob/master/stable/osclass/templates/_helpers.tpl